src/phys/dca_step/cluster_solver/shared_tools/accumulation/tp/tp_accumulator_kernels.cu

```diff
@@ -17,6 +18,7 @@
 #include <cuda.h>
```

Comment:
this files is missing #include <mpi.h> which the MPI_WORLD below requires.  In my build environment this breaks the build.

Reply:
Added, so does mci_parameters.hpp